README.md

```diff
@@ -88,7 +88,10 @@ a more lenient and customizable variant of JSON. Complete the following items in
     `CANVAS` | `API_CLIENT_SECRET` | The client secret for authenticating to the API Directory.
     `CANVAS` | `CANVAS_URL` | The Canvas instance URL to be used as the base URL for API requests that use the `CANVAS TOKEN`.
     `CANVAS` | `CANVAS_TOKEN` | The Canvas token used for authenticating to the API when not using the U-M API Directory.
-    `MIVIDEO` | `service_account_json_filename` | The name of the JSON credential file for accessing UDP's Google BigQuery service account.  This file name is appended to the value of `ENV_DIR` (which is `/config/secrets`, by default) to determine the full path to the file.<br/><br/>If this key's value is set to `umich-its-tl-reports-prod.json` and `ENV_DIR` has its default value, the full path to the file will be `/config/secrets/umich-its-tl-reports-prod.json`.
+    `MIVIDEO` | `udp_service_account_json_filename` | The name of the JSON credential file for accessing UDP's Google BigQuery service account.  It should be the `umich-its-tl-reports-prod.json` credential file for UMich ITS TL.  This file name is appended to the value of `ENV_DIR` (which is `/config/secrets`, by default) to determine the full path to the file.<br/><br/>If this key's value is set to `umich-its-tl-reports-prod.json` and `ENV_DIR` has its default value, the full path to the file will be `/config/secrets/umich-its-tl-reports-prod.json`.
+    `MIVIDEO` | `default_last_timestamp` | The MiVideo procedures use the last timestamp found in its tables in this application's DB to query for data newer than that time.  If that timestamp isn't found (e.g., the first time the application runs) the value of this property will be used.  This must be a valid ISO 8601 timestamp in the UTC time zone.  The recommended value is `2020-03-01T00:00:00+00:00`.
+    `MIVIDEO` | `kaltura_partner_id` | This is an integer that represents the Kaltura account number.  UMich ITS TL users can find this value in the usual security files folder.
+    `MIVIDEO` | `kaltura_user_secret` | This is a string that represents an administrator's key for the Kaltura account.  UMich ITS TL users can find this value in the usual security files folder.
```

Comment:
You'll need to add an entry for the new configuration variable for `Canvas_UMich`.